tests/integration_tests/charts/data/api_tests.py

```diff
@@ -425,6 +425,30 @@ def test_with_invalid_where_parameter__400(self):
 
         assert rv.status_code == 400
 
+    @pytest.mark.usefixtures("load_birth_names_dashboard_with_slices")
+    def test_with_invalid_where_parameter_closing_unclosed__400(self):
+        self.query_context_payload["queries"][0]["filters"] = []
+        # WHERE-clause closing unclosed parens
```

Comment:
redundant comment 
you should encapsulate the assignment into get method or constant who express the where instead using a comment